source/vocab/base.ttl

```diff
@@ -143,7 +143,7 @@ rdf:type a owl:ObjectProperty;
 
 :comment a owl:DatatypeProperty;
     rdfs:label "comment"@en, "kommentar"@sv;
-    sdo:domainIncludes :Agent, :Title, :ToCEntry ;
+    sdo:domainIncludes :Agent, :Title, :ToCEntry, :EAN, :UPC ;
```

Comment:
In the future, we might want to allow `:comment` on all `:StructuredValue`:s...

Reply:
Yes further discussions about usage is needed going forward.